src/server/standalone.js

```diff
@@ -852,24 +852,20 @@ function StandAloneServer(gmeConfig) {
                 logger.error('worker/simpleResult err', err);
                 res.sendStatus(500);
             } else {
-                res.header('Content-Disposition', 'attachment;filename=\'' + filename + '\'');
+                res.header('Content-Disposition', 'attachment;filename=' + filename);
```

Comment:
This will be wrong if filename contains `%` or non-ASCII. Use https://www.npmjs.com/package/content-disposition